plc4j/spi/src/main/java/org/apache/plc4x/java/spi/connection/AbstractPlcConnection.java

```diff
@@ -228,4 +230,14 @@ public CompletableFuture<PlcBrowseResponse> browseWithInterceptor(PlcBrowseReque
         return protocol.browseWithInterceptor(browseRequest, interceptor);
     }
 
+    @Override
+    public Optional<PlcTag> parseTagAddress(String tagAddress) {
+        PlcTag plcTag;
+        try{
+            plcTag = tagHandler.parseTag(tagAddress);
+        }catch (Exception e){
```

Comment:
the Exception should be at least logged at a debug level or if it never matters the variable should be renamed to `ignore`